lib/web_ui/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dev_dependencies:
   archive: 2.0.13
   http: 0.12.1
   image: 2.1.13
-  js: 0.6.1+1
+  js: 0.6.3-nullsafety.3
   mockito: 4.1.1
```

Comment:
Looking at the LUCI logs this creates an issue: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8864337262200012576/+/steps/pub_get_in_web_engine_tester__2_/0/stdout

Is it reproducible locally?